core/src/edu/asu/c3simulator/screens/MarketingWizard.java

```diff
@@ -0,0 +1,362 @@
+		campaignMethod = firstWizardPage.getSelectedMarketingMethod();
+		
+		// TODO add Corner Advisor
+		// TODO add Home Button
+		
+		NavigationPanel navigationPanel = createNavigationPanel();
+		
+		Table mainTable = createMainTable();
+		mainTable.setPosition(
+				stage.getWidth() - 0.01f * stage.getWidth() - mainTable.getWidth(),
+				0.01f * stage.getHeight());
+		
+		stage.addActor(navigationPanel);
+		stage.addActor(mainTable);
+	}
+	
+	/**
+	 * The method creates the navigation panel.
+	 * 
+	 * @return the NavigationPanel actor that was created.
+	 */
+	private NavigationPanel createNavigationPanel()
+	{
+		NavigationPanel navigationPanel = new NavigationPanel(game, skin);
+		
+		// TODO Check if all screens should be left null since leaving the wizard should
+		// only be done through the back button.
+		navigationPanel.addButton("Products", null);
+		navigationPanel.addSubButton("Products", "Pre-Market", null);
+		navigationPanel.addSubButton("Products", "Current", null);
+		navigationPanel.addSubButton("Products", "Retired", null);
+		navigationPanel.addButton("Product Growth", null);
+		navigationPanel.addSubButton("Product Growth", "Supply", null);
+		navigationPanel.addSubButton("Product Growth", "Demand", null);
+		navigationPanel.addButton("Employment", null);
+		navigationPanel.addButton("Marketing", null);
+		
+		navigationPanel.setPosition(0.01f * stage.getWidth(), stage.getHeight()
+				- (0.3f * stage.getHeight()));
+		
+		return navigationPanel;
+	}
+	
+	/**
+	 * This method creates the main section of the GUI
+	 * 
+	 * @return The table that was created.
+	 */
+	private Table createMainTable()
+	{
+		Table mainTable = new Table();
+		mainTable.setSize(0.80f * stage.getWidth(), 0.80f * stage.getHeight());
+		
+		Table upperSection = new Table();
+		Table lowerSection = new Table();
+		
+		Label campaignMethodLabel = new Label("Marketing Campaign: " + campaignMethod,
+				skin);
+		
+		TextButton campaignTargetLabel;
+		if (selectedProduct == null) // No product was selected. Thus, the company as a
+										// whole is the campaign target.
+			campaignTargetLabel = new TextButton("Promoting Company", skin);
+		else
+			campaignTargetLabel = new TextButton("Product: " + selectedProduct.getName(),
+					skin);
+		
+		campaignTargetLabel.setDisabled(true);
+		campaignTargetLabel.setColor(0.5f, 0.5f, 0.5f, 1.0f);
+		
+		TextButton backButton = new TextButton("Back", skin);
+		backButton.addListener(new ClickListener() {
+			@Override
+			public void clicked(InputEvent event, float x, float y)
+			{
+				game.setScreen(firstWizardPage);
+			}
+		});
+		
+		upperSection.add(backButton).center();
+		upperSection.add(campaignTargetLabel).expand().fill().center();
+		
+		Table campaignTypes = new Table();
+		Table campaignOptionTable = new Table();
+		Table productAndConsumerInformation = new Table();
+		
+		Label campaignTypesLabel = new Label("Campaign Types", skin);
+		List<String> campaignTypesList = new List<String>(skin);
```

Comment:
Reading this was a tad disorienting. perhaps use the namespace "ui" or add a comment indicating that this is not java.util.List